File: Hackerrank Questions/missingNumbers.java
import java.io.*;
class missingNumbers{
	public static void main(String[] args) {
		List<Integer> arr = {203,204,205,206,207,208,203,204,205,206};

		List<Integer> brr = {203,204,204,205,206,207,205,208,203,206,205,206,204};
		missingNumbers(arr,brr);
	}
	
	public static List<Integer> missingNumbers(List<Integer> arr , List<Integer> brr){
		List<Integer> result = new ArrayList<>();
		Map<Integer,Integer> map = new HashMap<>();

		for (Integer val : brr ) {
			map.put(val,map.getOrDefault(val,0)-1);
		}

		for (Integer val : arr ) {
			if (map.containsKey(val)) {
				map.put(val,map.getOrDefault(val,0) -1 );
			}
		}
        for(Map.Entry<Integer,Integer> entry:map.entrySet()){
            if(entry.getValue()>0){
                result.add(entry.getKey());
            }
        }
        return result;

	}
}

File: Hackerrank Questions/missingNumbers.cpp
#include <bits/stdc++.h>
using namespace std;

vector<int> missingNumbers(vector<int> arr, vector<int> brr) {
	
}

vector<int> inputVector(vector<int> arr, int length) {
	for (int i = 0; i < length; ++i)
	{
		int data;
		cin >> data;
		arr.push_back(data);
	}
	return arr ;
}

int main () {
	int n = 0 , m  = 0;
	cin >> n >> m;
	vector<int> arr;
	vector<int> brr ;
	inputVector(arr, n);
	inputVector(brr, m);
	missingNumbers(arr, brr);
}



